ui/standard-ui/src/main/java/com/ait/ui/standard/view/StandardUiFragment.kt

```diff
@@ -17,12 +22,19 @@ class StandardUiFragment : Fragment() {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View = FragmentStandardUiBinding.inflate(
-        inflater,
-        container,
-        false
-    ).also {
-        it.lifecycleOwner = viewLifecycleOwner
-        it.viewModel = viewModel
-    }.root
+    ): View  {
+        return FragmentStandardUiBinding.inflate(
```

Comment:
Why did you move assignment into inner block?
Code without ```return``` has one line less and the same readability

Reply:
Reverted